### PR TITLE
Add option to control how __init__.py is generated.

### PR DESCRIPTION
**Pull request by cherba**

### CODE REVIEW

File: apitools/gen/gen_client.py [hunk 1]
```diff
@@ -105,6 +105,7 @@ def _GetCodegenFromFlags(args):
         base_package=args.base_package,
         protorpc_package=args.protorpc_package,
         generate_cli=args.generate_cli,
+        init_wildcards_file=args.init_file == 'wildcards',
```
Comment (craigcitro) [inline apitools/gen/gen_client.py]: silly: parenthesize the expression on the RHS?

**Comment by craigcitro** [comment]
LGTM, one small nit and you're good to go

**Comment by cherba** [comment]
I signed it!

File: apitools/gen/gen_client.py (hunk 1 above)
Comment (cherba) [inline apitools/gen/gen_client.py]: Done.

**Pull request merged**